lib/base-compiler.js

```diff
@@ -57,6 +57,13 @@ Compile.prototype.newTempDir = function () {
             else
                 resolve(dirPath);
         });
+        if (process.env.winTmp != process.env.tmpDir && process.env.winTmp != "")
```

Comment:
Couldn't this be moved to the specific compiler types for wsl? If not, I'd love to know why (I can't seem to find a reason not to, so it would be great to know!) 😄

Reply:
This could, I think. There are two things that concern me about moving it to WSL-CL.js
1. If ever we have a second (or third!) Windows-only compiler, it'd have to exist in each.
2. This code appears to run once during setup, not during the main edit/compile loop. The code in WSL-CL runs during the loop. I don't know if that's a problem. I'll try moving it.